packages/shared/components/Button.svelte

```diff
@@ -49,7 +49,7 @@
         {form}
         class={`xl cursor-pointer text-center rounded-xl pt-8 pb-4 px-4 flex flex-col items-center ${classes}`}
         use:bindEvents={events}
-        on:click={onClick}
+        on:click|stopPropagation={onClick}
```

Comment:
perhaps we dont need this anymore as we are not using this component in the ActivityTile

Reply:
Why are we not using the button component for a button? 😬 😆

Reply:
Still probably wise otherwise we can have weird side affects in the future anyway